src/components/ChartSetting.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Menu from './Menu.jsx';
+import { connect } from '../store/Connect';
+import { 
+    EditIcon,
+    PositionLeftIcon,
+    PositionBottomIcon,
+    ThemeDarkIcon,
+    ThemeLightIcon,
+    ChevronRightIcon,
+    BackIcon,
+    FlagIcons
+} from './Icons.jsx';
+import '../../sass/_ciq-chart-setting.scss';
+
+
+const ChartSetting = ({
+    Menu,
+    menuOpen,
+    languages,
+    setView,
+    view,
+    setLanguage
+}) => {
+    const renderMain = () => {
+        return <div>
+            <div className='title'> {t.translate('Settings')} </div>
+            <div className='body'>
+                <div className="ciq-list-setting">
+                    {/*
+                        other chart setting option which not use for now
+                    <div className="ciq-item">
+                        <span className="ciq-icon-text">{t.translate('Position')}</span>
```

Comment:
I see you added some strings that need to be translated. You'll need to run `yarn translations` to add them to `messages.pot`. [This is documented in README](https://github.com/binary-com/SmartCharts#translations).

Since we are going to add Position and Theme settings eventually, uncomment these options before running translations and comment them out again.